src/actions/index.js

```diff
@@ -1,17 +1,20 @@
-import behindthename from '../apis/behindTheName';
-import axios from 'axios';
+import behindthename from "../apis/behindTheName";
+import axios from "axios";
+import { FETCH_NAMES, SIMILAR_NAMES, ERROR, CLEAR_ERRORS } from "./types";
 
 export const fetchNames = formValues => {
 	return async dispatch => {
 		const name = formValues.username;
-		const response = await axios.get(`https://www.behindthename.com/api/lookup.json?name=${name}&key=mo245439977`);
+		const response = await axios.get(
+			`https://www.behindthename.com/api/lookup.json?name=${name}&key=mo245439977`

```

Comment:
If that key parameter is your API key you'll probably want to extract that into a constant also.

Actually ideally it would be loaded as an environment variable, and all the repo would contain is a variable that gets set to whatever is loaded, so you don't have to commit the keys like this, avoids any ISIS interferance.

If you wanna set this up lmk and i'll go through it with you, it's important but luckily not that complicated.